packages/cli/src/commands/init/init.js

```diff
@@ -155,9 +165,20 @@ async function createFromReactNativeTemplate(
   }
 }
 
-function createProject(projectName: string, options: Options, version: string) {
-  fs.mkdirSync(projectName);
-  process.chdir(projectName);
+function createProject(
+  projectPath: string,
+  projectName: string,
+  options: Options,
+  version: string,
+) {
+  try {
+    fs.mkdirSync(projectPath, {recursive: true});
```

Comment:
Alternatives:
- use `mkdir -p` 
- manually split path by `path.delimiter` and call `mkdirSync` n times for each path chunk.

Reply:
We have `mkdirp` in the cli deps, so please use that for now

Reply:
Nice! I didn't remember this. I had a feeling there must be a JavaScript script that mimics this functionality, but quick Google didn't reveal anything.

Reply:
Yeah, noticed `mkdirp` in the deps today in the morning while working on #373, I'll change this :)_